scripts/DolbyIOManager.cs

```diff
@@ -24,6 +30,42 @@ public class DolbyIOManager : MonoBehaviour
         [Tooltip("Indicates if the DolbyIOManager should automatically close the session on application quit.")]
         public bool AutoCloseSession = true;
 
+        /// <summary>
+        /// For convenience during early development and prototyping, a GetToken (linked) method is provided for you to 
+        /// acquire a client access token directly from the application. However, please note Dolby does not recommend 
+        /// using this mechanism in the production software for (security best practices)[https://docs.dolby.io/communications-apis/docs/guides-client-authentication] reasons.
+        /// </summary>
+        /// <param name="key">The customer key.</param>
+        /// <param name="secret">The customer secret.</param>
+        /// <returns>An asynchronous task containing the token.</returns>
+        public static async Task<string> GetToken(string key, string secret)
+        {
+            string result = "";
+
+            using (var client = new HttpClient())
+            {
+                var request = new HttpRequestMessage(HttpMethod.Post, "https://api.dolby.io/v1/auth/token");
```

Comment:
We are still on https://session.voxeet.com/v1/oauth2/token for the client access token, this one is for REST API (api token).

Reply:
Ho! No problem